saleor/graphql/shop/tests/test_shop.py

```diff
@@ -1355,6 +1358,10 @@ def test_order_settings_update_by_staff(
     response_settings = content["data"]["orderSettingsUpdate"]["orderSettings"]
     assert response_settings["automaticallyConfirmAllNewOrders"] is False
     assert response_settings["automaticallyFulfillNonShippableGiftCard"] is False
+    assert (
+        response_settings["defaultTransactionFlowStrategy"]
+        == TransactionFlowStrategyEnum.CHARGE.name
+    )
```

Comment:
Shouldn't there be an assertion with the order settings field instead of a direct value?
I mean smth like this:
```suggestion
    assert (
        response_settings["defaultTransactionFlowStrategy"]
        == channel.default_transaction_flow_strategy
```